spec/index.html

```diff
@@ -791,9 +791,22 @@ <h3>Version Labels</h3>
       </tbody>
     </table>
 
-    <p>For serializations supporting in-line version announcement,
+    <p>
+      Data conforming to version "1.1" is valid as data with version "1.2-basic", 
+      and data conforming to version "1.2-basic" is valid as data with version "1.2".
+      Data conforming to version "1.1" has the same semantics 
+      under [[[RDF11-MT]]] and under [[[RDF12-SEMANTICS]]].
```

Comment:
In most similar docs I've worked on, when the triple-bracket is used (rendered as the title of the cited doc, with href of the external document), it's followed by a double-bracket (rendered as a cite of the doc, which may be normative or non, with href of the entry in the references section). I'm suggesting this only on this paragraph, to give you an idea, but my real suggestion is that it be applied throughout.
```suggestion
      under [[[RDF11-MT]]] [[RDF11-MT]] and under [[[RDF12-SEMANTICS]]] [[RDF12-SEMANTICS]].
```